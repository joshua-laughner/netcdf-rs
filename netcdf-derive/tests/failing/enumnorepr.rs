#[derive(netcdf_derive::NcType)]
enum NoRepr {
    A,
    B,
}

fn main() {}
