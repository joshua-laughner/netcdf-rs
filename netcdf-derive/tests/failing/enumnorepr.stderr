error: Can not derive NcType for enum without suitable repr
 --> tests/failing/enumnorepr.rs:2:1
  |
2 | / enum NoRepr {
3 | |     A,
4 | |     B,
5 | | }
  | |_^
  |
  = help: Add #[repr(i32)] (or another integer type) as an attribute to the enum
